pilot/pkg/model/validation.go

```diff
@@ -41,7 +41,7 @@ const (
 	dns1123LabelMaxLength int    = 63
 	dns1123LabelFmt       string = "[a-zA-Z0-9]([-a-z-A-Z0-9]*[a-zA-Z0-9])?"
 	// a wild-card prefix is an '*', a normal DNS1123 label with a leading '*' or '*-', or a normal DNS1123 label
-	wildcardPrefix string = `\*|(\*|\*-)?(` + dns1123LabelFmt + `)`
+	wildcardPrefix string = `(\*|(\*|\*-)?` + dns1123LabelFmt + `)`
```

Comment:
It is not clear how this change fixes things... as I don't see us not allow `/` here.

Reply:
`(` was in wrong place, so the regex didn't work as expected

Reply:
Thanks, I missed wildcardPrefixRegexp() line 65.